src/game_api/script/lua_vm.cpp

```diff
@@ -254,6 +254,7 @@ end
         return backend->cbcount++;
     };
     /// Clear previously added callback `id` or call without arguments inside any callback to clear that callback after it returns.
+    // lua["clear_callback"] = [](sol::optional<CallbackId> id) -> void {};
```

Comment:
Would probably be better to just make two separate functions for this and save overload to variable as we always do